perun-wui-profile/src/main/java/cz/metacentrum/perun/wui/profile/pages/personal/PersonalView.java

```diff
@@ -42,6 +49,14 @@
 	private static final String PREFERRED_LANGUAGE = "preferredLanguage";
 	private static final String TIME_ZONE = "timezone";
 
+	private static final Map<String, String> attributeNamesTranslations = new HashMap<>();
+
+	private static final String URN_ORGANIZATION = "urn:perun:user:attribute-def:def:organization";
+	private static final String URN_PREFERRED_LANGUAGE = "urn:perun:user:attribute-def:def:preferredLanguage";
+	private static final String URN_PREFERRED_EMAIL = "urn:perun:user:attribute-def:def:preferredMail";
+	private static final String URN_TIMEZONE = "urn:perun:user:attribute-def:def:timezone";
+	private static final String SPECIAL_FULLNAME = "special:user:fullName";
```

Comment:
Since you are actually getting ``displayName`` param of ``User`` object, you can use "core" attributes for such task instead of faking special attribute. Here it would be ``urn:perun:user:attribute-def:core:displayName``. You can pass such value to getAttributes methods which takes list of urns.